### DISCUSSION TITLE
Save issue

### DISCUSSION THREAD

**Discussion by neet90** [URUWorks/TeroSubtitler#292] [Q&A]
Tero Subtitler v1.0.1.7

Hello,

My probleme here is when you want to save the subtitle with the translator mode, the translate text crush the transcript text. Also, I can't use saved project and sometimes I can't use saved subtitle and I have an error message : "error loading unreconized format".

I hope someone can say I only misused the app..

**Comment by URUWorks** [comment]
Hola, 

Can you explain better and if it is possible with an example video?

**Reply by neet90** [reply]
Thanks for the reply,

I just can't open previously saved subtitle files and project files, I hope that the video will help.

I precise that I am on Mac Ventura 13.6.5

https://github.com/URUWorks/TeroSubtitler/assets/164443832/fcff61d4-bffd-474c-831f-a24b50d9acda

**Comment by URUWorks** [comment]
Can you confirm if the latest version still does not open said file?

**Reply by neet90** [reply]
Yes the problem is still there

**Comment by URUWorks** [comment]
Can you send us the file in question via discord?

**Comment by URUWorks** [comment]
fixed in v1.0.1.9
